packages/core/src/Button/index.tsx

```diff
@@ -497,3 +498,174 @@ export const IconButton = React.forwardRef<BaseElement, IconButtonProps>(
     )
   }
 )
+
+/**
+ * IconTextButton
+ *
+ * Always has a primary icon and a secondary text
+ *
+ */
+
+export const NativeIconTextButton = styled.button<{
+  readonly icon: IconType
```

Comment:
Do you use `icon` in this styled component?

Reply:
Removed.